docs/readthedocs/source/doc/Nano/Tutorials/seq_and_func.ipynb

```diff
@@ -0,0 +1,574 @@
+      "outputs": [],
+      "source": [
+        "from bigdl.nano.automl.hpo.visualization import plot_optimization_history\n",
+        "plot_optimization_history(study)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 542
+        },
+        "id": "QdZhgElEmexz",
+        "outputId": "3da0abb3-bc1b-4412-8834-0e995ea27857"
+      },
+      "outputs": [],
+      "source": [
+        "from bigdl.nano.automl.hpo.visualization import plot_parallel_coordinate\n",
+        "plot_parallel_coordinate(study)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 542
+        },
+        "id": "rL8-6vGYmex0",
+        "outputId": "c7ac5ec6-2af9-4b9d-9815-e98b4466d732"
+      },
+      "outputs": [],
+      "source": [
+        "from bigdl.nano.automl.hpo.visualization import plot_intermediate_values\n",
+        "plot_intermediate_values(study)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 542
+        },
+        "id": "A5mqqXg8mex0",
+        "outputId": "325accdc-c371-48d1-d9e2-ce332759d9bb"
+      },
+      "outputs": [],
+      "source": [
+        "from bigdl.nano.automl.hpo.visualization import plot_contour\n",
+        "plot_contour(study)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 542
+        },
+        "id": "MUmsUyAfmex0",
+        "outputId": "ea74030c-309f-468f-80e6-718956a44cc2"
+      },
+      "outputs": [],
+      "source": [
+        "from bigdl.nano.automl.hpo.visualization import plot_param_importances\n",
+        "plot_param_importances(study)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 542
+        },
+        "id": "Y-oXajiTmex1",
+        "outputId": "e678b912-7946-41c1-e431-8e61e9a28dca",
+        "scrolled": true
+      },
+      "outputs": [],
+      "source": [
+        "plot_param_importances(study, target=lambda t: t.duration.total_seconds(), target_name=\"duration\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "You can find the running output from [here](https://github.com/intel-analytics/BigDL/blob/main/python/nano/notebooks/hpo/seq_and_func.ipynb), or run the notebook by yourself in [Google Colab](https://colab.research.google.com/github/intel-analytics/BigDL/blob/main/python/nano/notebooks/hpo/seq_and_func.ipynb)."
```

Comment:
I added these two sentences to navigate the readers to the github and colab.

However, I ran the notebook on google colab, the interactive figures cannot be shown there either.